master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -43,7 +43,7 @@ spec:
 | logSeveritySys | The log severity above which logs are sent to the syslog. Set to `NONE` for no logging to syslog. | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) | string | `INFO` |
 | logSeverityFile| The log severity above which logs are sent to the log file. | Same as `logSeveritySys` | string | `INFO` |
 | logSeverityScreen | The log severity above which logs are sent to the stdout. | Same as LogSeveritySys | string | `INFO` |
-| ignoreLooseRPF\* | Set to `true` to allow Felix to run on systems with loose reverse path forwarding (RPF). | boolean | boolean | `false` |
+| ignoreLooseRPF | Set to `true` to allow Felix to run on systems with loose reverse path forwarding (RPF). **Warning**: Setting `ignoreLooseRPF` to `true` increasing the risk of spoofing as a workload endpoint can use the IP address of another workload endpoint as its source IP address. For containers, you can mitigate this threat by adjusting the permissions to prevent arbitrary source IPs. | boolean | boolean | `false` |
```

Comment:
typo: `increasing`

> For containers, you can mitigate this threat by adjusting the permissions to prevent arbitrary source IPs
The default is to have an unprivileged container, which shouldn't be able to do this so I think it's probably better to turn this around and warn about giving privileges, rather than suggesting taking them away.

WDYT to this: 
```
**Warning**: Calico relies on "strict" RPF checking being enabled to prevent workloads, such as VMs and privileged containers, from spoofing their IP addresses and impersonating other workloads (or hosts).  Only enable this flag if you need to run with "loose" RPF and, either: you trust your workloads, or, you have another mechanism in place to prevent spoofing.
```

Reply:
All right great, thanks for the clarification and suggestion.